torch/csrc/THP.h

```diff
@@ -10,10 +10,14 @@
 // define PyInt_* macros for Python 3.x.  NB: We must include Python.h first,
 // otherwise we'll incorrectly conclude PyInt_Check isn't defined!
 #ifndef PyInt_Check
-#define PyInt_Check             PyLong_Check
-#define PyInt_FromLong          PyLong_FromLongLong
-#define PyInt_AsLong            PyLong_AsLong
-#define PyInt_Type              PyLong_Type
+# define PyInt_Check            PyLong_Check
+# ifdef _WIN32
+#  define PyInt_FromLong        PyLong_FromLongLong
+# else
+#  define PyInt_FromLong        PyLong_FromLong
```

Comment:
Hum... I might be missing something in here, but before we had
```c
#define PyInt_FromLong          PyLong_FromLongLong
```
for unix systems, now we have
```c
#define PyInt_FromLong        PyLong_FromLong
```
Maybe I'm missing something here?